boards/common/nrf52/bootloader_nrfutil.dep.mk

```diff
@@ -1,6 +1,6 @@
 include $(RIOTBOARD)/common/makefiles/stdio_cdc_acm.dep.mk
 
 # enable bootloader reset over USB, requires CDC ACM to be used
-ifneq (,$(filter stdio_cdc_acm,$(USEMODULE)))
+ifneq (,$(filter stdio_cdc_acm stdio_tinyusb_cdc_acm,$(USEMODULE)))
```

Comment:
The reset via CDC ACM abuses the fact that CDC ACM abstracts modem like devices, which include the ability to set the symbol rate (often incorrectly referred to as "baudrate"). Changing the symbol rate makes no sense in the RIOT context, but is abused as side channel to trigger the reset.

As far as I know, this hack is not implemented in TinyUSB. Can you double check that it is indeed provided?

Reply:
I have indeed checked that resetting the board via usb works even with tinyusb with this change. There's also `pkg/tinyusb/contrib/tinyusb_board_reset.c` which seems to implement that.

Reply:
@maribu for the record, I think this is the relevant [PR](https://github.com/RIOT-OS/RIOT/pull/19006)